webpack.config.js

```diff
@@ -12,10 +12,8 @@ module.exports = (env, argv) => {
     entry: "./src/index.ts",
     mode: "development",
     output: {
-      publicPath: isProduction
-        ? undefined
-        : // Required for the historyApiFallback setting to work per https://ui.dev/react-router-cannot-get-url-refresh#webpack--development
-          `http://localhost:${devPort}/`,
+      // Required for the historyApiFallback setting to work per https://ui.dev/react-router-cannot-get-url-refresh#webpack--development
+      publicPath: "/remote2/",
```

Comment:
I don’t think we need this. I think this is related to a bad config.

Reply:
This was required for me to get it working with the Kubernetes Nginx Ingress controller.  I will investigate further.  I most likely need to handle a rewrite differently with the Ingress object.